game/match.py

```diff
@@ -1,16 +1,15 @@
 #!/usr/bin/env python
 from gameboard import GameBoard
-from game.player_with_strategy_one import PlayerWithStrategyOne
-from game.player_with_strategy_two import PlayerWithStrategyTwo
+from game.player import Player
 from itertools import cycle
 
 class GameIsOver(Exception): # Where should this exception be placed?
     def __init___(self, winner):
         self.winner = winner
 
 class Match(object):
-    def __init__(self, player_one = PlayerWithStrategyOne('red'),
-                 player_two = PlayerWithStrategyTwo('blue'), board = GameBoard()):
+    def __init__(self, player_one = Player('red'),
```

Comment:
I just created #15 to keep track of that, since I think this issue is out of the scope of the PR